pkg/commands/git_commands/commit.go

```diff
@@ -178,3 +177,12 @@ func (self *CommitCommands) RevertMerge(sha string, parentNumber int) error {
 func (self *CommitCommands) CreateFixupCommit(sha string) error {
 	return self.cmd.New(fmt.Sprintf("git commit --fixup=%s", sha)).Run()
 }
+
+func (self *CommitCommands) GetCommitMessageFromHistory(value int) (string, error) {
+	hash, _ := self.cmd.New(fmt.Sprintf("git log -1 --skip=%d --pretty=%%H", value-1)).DontLog().RunWithOutput()
```

Comment:
`git log` fails when a repository is empty.

```suggestion
	hash, _, err := self.cmd.New(fmt.Sprintf("git log -1 --skip=%d --pretty=%%H", value-1)).DontLog().RunWithOutputs()
	if err != nil {
		return "", ErrInvalidCommitIndex
	}
```